src/main/java/edu/cornell/kfs/concur/batch/report/ConcurStandardAccountingExtractBatchReportData.java

```diff
@@ -0,0 +1,142 @@
+        this.cashAdvancesRelatedToExpenseReports = cashAdvancesRelatedToExpenseReports;
+        this.expensesPaidOnCorporateCard = expensesPaidOnCorporateCard;
+        this.transactionsBypassed = transactionsBypassed;
+        this.pdpRecordsProcessed = pdpRecordsProcessed;
+        this.validationErrorFileLines = validationErrorFileLines;
+        this.pendingClientObjectCodeLines = pendingClientObjectCodeLines;
+    }
+    
+    public String getConcurFileName() {
+        return concurFileName;
+    }
+
+    public void setConcurFileName(String concurFileName) {
+        this.concurFileName = concurFileName;
+    }
+
+    public ConcurBatchReportHeaderValidationErrorItem getHeaderValidationErrors() {
+        return headerValidationErrors;
+    }
+
+    public void setHeaderValidationErrors(ConcurBatchReportHeaderValidationErrorItem headerValidationErrors) {
+        this.headerValidationErrors = headerValidationErrors;
+    }
+
+    public ConcurBatchReportSummaryItem getReimbursementsInExpenseReport() {
+        return reimbursementsInExpenseReport;
+    }
+
+    public void setReimbursementsInExpenseReport(ConcurBatchReportSummaryItem reimbursementsInExpenseReport) {
+        this.reimbursementsInExpenseReport = reimbursementsInExpenseReport;
+    }
+
+    public ConcurBatchReportSummaryItem getCashAdvancesRelatedToExpenseReports() {
+        return cashAdvancesRelatedToExpenseReports;
+    }
+
+    public void setCashAdvancesRelatedToExpenseReports(ConcurBatchReportSummaryItem cashAdvancesRelatedToExpenseReports) {
+        this.cashAdvancesRelatedToExpenseReports = cashAdvancesRelatedToExpenseReports;
+    }
+
+    public ConcurBatchReportSummaryItem getExpensesPaidOnCorporateCard() {
+        return expensesPaidOnCorporateCard;
+    }
+
+    public void setExpensesPaidOnCorporateCard(ConcurBatchReportSummaryItem expensesPaidOnCorporateCard) {
+        this.expensesPaidOnCorporateCard = expensesPaidOnCorporateCard;
+    }
+
+    public ConcurBatchReportSummaryItem getTransactionsBypassed() {
+        return transactionsBypassed;
+    }
+
+    public void setTransactionsBypassed(ConcurBatchReportSummaryItem transactionsBypassed) {
+        this.transactionsBypassed = transactionsBypassed;
+    }
+
+    public ConcurBatchReportSummaryItem getPdpRecordsProcessed() {
+        return pdpRecordsProcessed;
+    }
+
+    public void setPdpRecordsProcessed(ConcurBatchReportSummaryItem pdpRecordsProcessed) {
+        this.pdpRecordsProcessed = pdpRecordsProcessed;
+    }
+
+    public List<ConcurBatchReportLineValidationErrorItem> getValidationErrorFileLines() {
+        return this.validationErrorFileLines;
+    }
+
+    public void setValidationErrorFileLines(List<ConcurBatchReportLineValidationErrorItem> validationErrorFileLines) {
+        this.validationErrorFileLines = validationErrorFileLines;
+    }
+
+    public void addErroringValidationFileLine(ConcurBatchReportLineValidationErrorItem validationErrorFileLine) {
```

Comment:
Should this be "addValidationErrorFileLine" instead of "addErroringValidationFileLine"?